src/containers/form/AppActionFactory.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * @flow
+ */
+
+import { newRequestSequence } from '../../core/redux/RequestSequence';
```

Comment:
I've extracted `RequestSequence` into its own library, with several improvements:

https://github.com/openlattice/redux-reqseq

Please upgrade to `redux-reqseq`. I'll make comments on what needs to be updated.